README.md

```diff
@@ -87,3 +87,26 @@ yarn rebuild
 ```
 
 Rebuild a fresh local instance of your site. Imports the canonical database backup and imports configuration into it.
+
+## Semantic Versioning
+
+Setup
+-----
+
+  1. This repo has the following named/maintenance branches:
+```
+main
+x.x
+x.x.x
+```
+  2. These branches are protected on GitHub
+  3. A personal access token was created for CircleCI.
+  4. CircleCI was setup to run on this project and tag the releases
+  5. Commit changes following the [Angular commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines)
```

Comment:
You're using the [`conventional` commit standard](https://www.conventionalcommits.org/en/v1.0.0/) rather than the Angular one (as per your commitlint.config.js configuration)

Reply:
@patrickocoffeyo Do you have thoughts on which one we should be using?  Recommended fix here?

Reply:
The conventional standard is the one I recommend. Angular enforces rules (such as requiring a topic) that are somewhat prohibitive. @generalredneck may have additional thoughts on this :)

Reply:
Ok, so I just need to update the readme, is what it sounds like.

Reply:
Readme update to use conventional commit standard.